src/style/loqui/index.css

```diff
@@ -1651,6 +1655,18 @@ article.form ul.listBox li.add:before {
     border: none;
     border-radius: 0;
   }
+  section#chat #footbox #typing {
+    position: fixed;
+    z-index: 9;
+    bottom: 0.1rem;
+    top: auto;
+  }
+  button {
+    z-index: 9;
+  }
+  section#chat nav#plus {
+    display: none;
+  }
```

Comment:
Yes in landscape. It's inside the media query. Buttons wont work with the new fixed footbox in landscape without higher z-index, at least on my device.  loqui/im schrieb:In src/style/loqui/index.css:
> @@ -1651,6 +1655,18 @@ article.form ul.listBox li.add:before {
>      border: none;
>      border-radius: 0;
>    }
> +  section#chat #footbox #typing {
> +    position: fixed;
> +    z-index: 9;
> +    bottom: 0.1rem;
> +    top: auto;
> +  }
> +  button {
> +    z-index: 9;
> +  }
> +  section#chat nav#plus {
> +    display: none;
> +  }


typing: you're talking of landscape mode?
button: I tried the plus button in landscape mode, it works for me...


—Reply to this email directly or view it on GitHub.